packages/core/src/components/popover-core/popover-core.tsx

```diff
@@ -277,9 +273,11 @@ export class TdsPopoverCore {
   }
 
   render() {
+    console.log('fade is this---> ', this.animation);
+
     const classes = {
+      'tds-popover-fade-animation': this.animation === 'fade',
       'is-shown': this.isShown,
```

Comment:
I think it is better to build a string based on the animation name, since there might be other animations added down the line.
```suggestion
    const classes = {
      'tds-popover-fade-${this.animation}': true,
      'is-shown': this.isShown,
```